src/actions/auth.js

```diff
@@ -63,7 +63,7 @@ function getProfileFromSnap(snap) {
   }
   // Firestore
   if (snap && snap.data && snap.exists) {
-    return snap.data()
+    return { ...snap.data(), uid: snap.id }
```

Comment:
This may be considered a breaking change since we are changing the return type of this function which I believe returns values to a public facing function (i.e. changing the external API).